### ISSUE TITLE
Create Campaign - Image Upload Limits

### ISSUE THREAD

**Issue by stevejgordon** [HTBox/allReady#2219]
When logged in as a site admin or organization admin, the user can create new campaigns in allReady.

Currently there is an image upload option but we do not limit the size of the upload or scale the uploaded image. This may lead to unneccesarily large images being uploaded and affecting page load times.

We should do at least one of the following:
1) Limit the max dimension on the long edge for the image and/or the filesize
2) Ideally, scale the image or provide a client side image cropper and resizer before uploading the image.

**Comment by stevejgordon** [comment]
Reserved for .NET South East codeathon

**Comment by Paul-Hadfield** [comment]
I'll pick this up after #2228 as code has been created for that issue

**Comment by Paul-Hadfield** [comment]
I think this was has been fixed and waiting to be merged into main code base.

**Comment by stimms** [comment]
@stevejgordon or @MisterJames can you confirm that this issue is resolved?
